alerter.cpp

```diff
@@ -1,32 +1,62 @@
 #include <iostream>
 #include <assert.h>
+#include <string>
 
 int alertFailureCount = 0;
+#define TEMPTHRESHOLD 200.0
+#define SUCCESS 200
+#define FAILURE 500
 
-int networkAlertStub(float celcius) {
+int testNetworkAlertStub(float celcius)

```

Comment:
Stub code should not be part of production code. all test related code such as test cases and test stubs should be segregated separately so that deployment does not contain these. unexpected execution of this code in production environment poses greater problems.